common/crypto_utils/avalon_crypto_utils/crypto_utility.py

```diff
@@ -252,10 +252,10 @@ def byte_array_to_hex(byte_array):
 
 
 # -----------------------------------------------------------------------------
-def compute_message_hash(message):
+def compute_message_hash(message_bytes):
```

Comment:
As long as you are changing this, add a docstring. Something like this (modified from crypto_utils.cpp):

```
"""
Compute SHA256 hash of message_bytes.

Parameters:
    message_bytes Data to hash
Returns:
    binary hash of data
"""
```

Reply:
Done